SocialFilter/sections/specific.txt

```diff
@@ -6487,7 +6486,7 @@ mobil.ksta.de##.social_btn
 znaj.ua##.field-name-addtoany
 connect-trojan.net###floating-fblog
 gismeteo.ua,gismeteo.by,gismeteo.kz,gismeteo.pl,gismeteo.com,gismeteo.lt,gismeteo.ru##.article__ctrls
-ibb.co,extraimage.info,rintor.space,lookimg.com,gifyu.com,screenshot.ru##.panel-share-networks
+imagehousing.com,picklik.ru,ibb.co,extraimage.info,rintor.space,lookimg.com,gifyu.com,screenshot.ru##.panel-share-networks
```

Comment:
It is a time to make it generic. Many sites uses same engine:
https://publicwww.com/websites/%22panel-share-networks%5C%22%22/